idrstream/cellprofiler_metadata.py

```diff
@@ -0,0 +1,20 @@
+"""
+This function will convert the `data_to_process.tsv` file into a .csv so that CellProfiler can read in the metadata and connect the appropriate
+metadata to the images from each batch. 
+"""
+import pandas as pd
+
+def convert_tsv_to_csv(data_to_process_tsv: str):
+    """ Convert `data_to_process.tsv` file into a .csv for CellProfiler to read in when processing the images
+
+    Parameters
+    ----------
+        data_to_process_tsv : str
+            string path to the `data_to_process.tsv` file
+    """
+    # read in metadata tsv file
+    data_to_process_csv = pd.read_table(data_to_process_tsv,sep='\t')
```

Comment:
Is there a reason to use `read_table` here instead of `read_csv`? If not, I think it would be better to stick with `read_csv` for consistency with the rest of the repository.

Reply:
I found some issues with `read_csv` when creating one of my functions in `idr_stream.py` but I can change it since it shouldn't cause issues here with just loading in then converting.